assets/components/AuditInformation.jsx

```diff
@@ -48,7 +48,7 @@ class AuditInformation extends React.Component {
                 <div className='wire-column__preview__date'>
                     {gettext('Updated {{updator}}at {{updated}}',
                         {
-                            updator: updator ? gettext(`by ${updator.first_name} ${updator.last_name} `) : 'by System ',
+                            updator: updator ? gettext(`by ${updator.first_name} ${updator.last_name} `) : gettext('by System '),
```

Comment:
imo the space shouldn't be part of the gettext call, that might get removed on transifex